src/components/node/stanford-page/home-page/today-hours.tsx

```diff
@@ -80,6 +84,9 @@ const LibrariesTodayHours = ({libraries, ...props}: { libraries: HoursProps["lib
               {library?.suLibraryHours &&
                 <TodayLibraryHours branchId={library.suLibraryHours}/>
               }
+              {library?.map &&
+                <span className="float-right w-auto inline"><span className="sr-only">{library.title} Location</span><Link href={library?.map} aria-label="Link to map"><MapPinIcon title="Map" width={25} className="mr-5 inline"/>Location</Link></span>
```

Comment:
the `sr-only` needs to be within the `<Link>` also you can remove the `aria-label` property.

Reply:
Ok this is fixed.